actionpack/test/controller/rescue_test.rb

```diff
@@ -131,22 +131,6 @@ def resource_unavailable_raise_as_string
   def missing_template
   end
 
-  def io_error_in_view
-    begin
-      raise IOError.new('this is io error')
-    rescue
-      raise ActionView::TemplateError.new(nil)
-    end
-  end
-
-  def zero_division_error_in_view
-    begin
-      raise ZeroDivisionError.new('this is zero division error')
-    rescue
-      raise ActionView::TemplateError.new(nil)
-    end
-  end
-
```

Comment:
Superfluous tests. Covered by `rescue when wrapper has more specific handler than cause` and friends.